substra/sdk/backends/remote/backend.py

```diff
@@ -109,26 +113,99 @@ def add(self, spec, exist_ok=False, spec_options=None):
                 exist_ok=exist_ok,
                 spec_options=spec_options,
             )
+        elif asset_type == schemas.Type.ComputePlan and spec_options.pop(AUTO_BATCHING, False):
+            assert batch_size
```

Comment:
Could you replace this with an exception?

Reply:
Yes, I'll do it. I want to point out that this is an error for the dev though, not the user